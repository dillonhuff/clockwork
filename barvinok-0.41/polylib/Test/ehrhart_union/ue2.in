
4 5
1 -1 0 1 0
1 0 1 0 0
1 1 -1 0 0
1 0 -1 0 10

4 5
1  1  0  0 -4
1 -1  0  1  0
1  0  1  0 -4
1  0 -1  1  0

4 5
1  1  0  1 -7
1 -1  0  0  7
1  1  1  0 -7
1 -1 -1  1  7



1 3
1 1 0		# P>=0


N
test


#####
# 4 domaines disjoints à trouver au début :

4 5
   1   -1    0    1    0 
   1    0    1    0    0 
   1    1   -1    0    0 
   1    0   -1    0    3 

5 5
   1    1    0    0   -4 
   1    0   -1    1    0 
   1   -1    1    0   -1 
   1    0    0    0    1 
   1    0   -1    0   10 

5 5
   1    1    0    0   -4 
   1   -1    0    1    0 
   1    0   -1    1    0 
   1    0    1    0  -11 
   1    0    0    0    1 

4 5
   1    1   -1    0    0 
   1    0   -1    0   10 
   1   -1    0    1    0 
   1    0    1    0   -4 

#####
#




