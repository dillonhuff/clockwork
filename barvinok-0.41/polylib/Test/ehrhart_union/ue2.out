         N  >= 0
         - N + 3 >= 0

( 3/2 * N^2 + 7/2 * N + 2 )

-----------------------------------
         N  -8 = 0
          1 >= 0

( 3/2 * N^2 + 7/2 * N + -18 )

-----------------------------------
         - N + 10 >= 0
         N  -9 >= 0

( 2 * N^2 + -4 * N + 10 )

-----------------------------------
         N  -11 >= 0
          1 >= 0

( 2 * N^2 + -4 * N + 10 )

-----------------------------------
         N  -7 = 0
          1 >= 0

( 2 * N^2 + -8 * N + 38 )

-----------------------------------
         - N + 6 >= 0
         N  -5 >= 0

( 1/2 * N^2 + 17/2 * N + -4 )

-----------------------------------
         N  -4 = 0
          1 >= 0

( 0 * N^2 + 12 * N + -10 )

-----------------------------------
         N  -6 = 0
          1 >= 0

( 1 * N^2 + 2 * N + 17 )

-----------------------------------
